website/docs/components/accordion/partials/guidelines/guidelines.md

```diff
@@ -96,13 +96,13 @@ Use the `flush` variant where space is limited such as within a [Card](/componen
 
 ### Toggle type
 
-The toggle accepts many different types of content, from text-based content to nested HDS components or custom content.
+The toggle accepts many different types of content, from text-based content to nested HDS components or generic content.
```

Comment:
(question, non-blocking) apologies if we have already discussed it and I've missed it; but did we discuss that a [generic](https://www.typescriptlang.org/docs/handbook/2/generics.html) is an existing term in TypeScript? Are we okay with overloading the term?

Reply:
@MelSumner I don't feel like I can 100% answer this correctly, so I'm going to loop in @heatherlarsen here 🙂

Reply:
(Not Heather), I see the point but I don't necessarily see this as a problem because:

- The tooling is fundamentally different, this specific method is specific to Figma.
- Designers probably have very little working/tactical knowledge about TypeScript. And honestly If I understand the purpose the purpose of a generic in TypeScript, there's overlap conceptually between the two, which actually might be a value add.

Reply:
Hmm, that's a good point @MelSumner. I didn't realize that. For context, the reason we're making this change is because our code is already using the term "generic" instead of "custom." By changing the terminology in Figma, we're not introducing a breaking change, while changing the terminology in code would result in a breaking change. 

cc @didoo do you have concerns about this?

Reply:
I think the `content` word after `generic` makes clear what we're referring to.